kustomize/components/without-loadgenerator/delete-loadgenerator.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: loadgenerator
+$patch: delete
```

Comment:
I've renamed this file from `remove-loadgenerator.patch.yaml` to `delete-loadgenerator.yaml`.
**Rationale for change:** 
* `delete` instead of 
* I like `*.patch.*`, but if `*.patch.*` is a convention we should follow, we should apply it to all other Kustomize components in this repo. I've removed it for now, for consistency.

Reply:
I am OK with delete although it might be misleading about time it is "deleted". I think the term is not right here, but it makes sense to follow kustomize patch syntax.
I disagree about *.patch.yaml. The reasons are:
1. Following the not appropriate approach just because older content follows it is not a good argument. Quite an opposite, new changes should follow the recommended processes and patterns and modify all old content each time it is touched.
2. Existing content does not use external files for patches.

Reply:
> new changes should follow the recommended processes and patterns and modify all old content each time it is touched.
Ah, that's a good point.
I'll unresolve this conversation.
Please feel free to bring back `*.patch.yaml`. Totally up to you.
We can make a mental note to follow the standard for now.

> might be misleading about time it is "deleted".
Good point. I agree. But I guess that's a Kustomize issue (`$patch: delete` in misleading).